index.js

```diff
@@ -25,6 +25,19 @@ module.exports.onRendererWindow = async (window) => {
   if (await isElevated()) {
     window.document.documentElement.classList.add('elevated');
   }
+
+  var config = window.config.getConfig();  
```

Comment:
`let` is much safer than using `var` [article on the subject](https://medium.com/craft-academy/javascript-variables-should-you-use-let-var-or-const-394f7645c88f)

I also think your `config` can use `const`

Reply:
We should probable add [no-var](https://eslint.org/docs/rules/no-var) and [prefer-const](https://eslint.org/docs/rules/prefer-const) to the `eslint.json` config